Upstream patch from https://www.mpfr.org/mpfr-current/#fixed
Prepated for EasyBuild by Simon Branford (Uni. of Birmingham)
diff -Naurd mpfr-4.0.2-a/src/mpfr-impl.h mpfr-4.0.2-b/src/mpfr-impl.h
--- mpfr-4.0.2-a/src/mpfr-impl.h    2019-01-27 18:30:16.000000000 +0000
+++ mpfr-4.0.2-b/src/mpfr-impl.h    2019-06-02 17:05:36.145226719 +0000
@@ -57,6 +57,7 @@
 
 #include <stdlib.h>
 #include <limits.h>
+#include <float.h>  /* for FLT_RADIX, etc., tested below */
